parley/src/layout/mod.rs

```diff
@@ -41,10 +41,18 @@ pub use run::RunMetrics;
 #[derive(Copy, Clone, Default, PartialEq, Eq, Debug)]
 #[repr(u8)]
 pub enum Alignment {
+    /// This is [`Alignment::Left`] for LTR text and [`Alignment::Right`] for RTL text.
     #[default]
     Start,
-    Middle,
+    /// This is [`Alignment::Right`] for LTR text and [`Alignment::Left`] for RTL text.
     End,
+    /// Align content to the left edge.
+    Left,
```

Comment:
I'd probably want to strongly encourage users to use `Alignment::Start` instead. They're more likely to get reasonable bidi handling if they do.